### DISCUSSION TITLE
export 'Navbar' (imported as 'Navbar') was not found in 'flowbite-react'

### DISCUSSION THREAD

**Discussion by iamaanahmad** [themesberg/flowbite-react#1399] [Q&A]
After deploying my project this errors are showing:
Compiled with problems:
×
ERROR in ./src/admin/admins.js 48:44-50
export 'Navbar' (imported as 'Navbar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/admins.js 51:46-58
export 'Navbar' (imported as 'Navbar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/billing.js 37:44-50
export 'Navbar' (imported as 'Navbar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/billing.js 40:46-58
export 'Navbar' (imported as 'Navbar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/cancellation.js 37:44-50
export 'Navbar' (imported as 'Navbar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/cancellation.js 40:46-58
export 'Navbar' (imported as 'Navbar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminhead.js 34:30-36
export 'Navbar' (imported as 'Navbar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminsidebar.js 63:30-37
export 'Sidebar' (imported as 'Sidebar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminsidebar.js 72:35-48
export 'Sidebar' (imported as 'Sidebar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminsidebarmobile.js 65:30-37
export 'Sidebar' (imported as 'Sidebar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminsidebarmobile.js 75:35-48
export 'Sidebar' (imported as 'Sidebar') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminstable.js 48:37-42
export 'Table' (imported as 'Table') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminstable.js 49:40-50
export 'Table' (imported as 'Table') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminstable.js 51:42-56
export 'Table' (imported as 'Table') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminstable.js 58:41-55
export 'Table' (imported as 'Table') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminstable.js 65:41-55
export 'Table' (imported as 'Table') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminstable.js 77:39-49
export 'Table' (imported as 'Table') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminstable.js 79:60-69
export 'Table' (imported as 'Table') was not found in 'flowbite-react' (possible exports: theme)
ERROR in ./src/admin/components/adminstable.js 81:44-54
export 'Table' (imported as 'Table') was not found in 'flowbite-react' (possible exports: theme)

**Comment by SutuSebastian** [comment]
There has been a lot of breaking changes since, and from version `0.11.x` things improved.

Do u still get this issue when using the latest version?

**Reply by iamaanahmad** [reply]
Resolved, Thanks!
